src/Toast/README.md

```diff
@@ -11,85 +12,88 @@ devStatus: 'Done'
 notes: ''
 ---
 
-``Toast`` is a pop-up style message that shows the user a brief, fleeting, dismissible message about a successful app process.
+`Toast` is a pop-up style message that shows the user a brief, fleeting, dismissible message about a successful app process.
 
-``Toasts`` sit fixed to the bottom left of the window.
+## Features
+
+- **Customizable Appearance**: Choose the window position for toast.
+- **Interactive**: Includes a close button for manual dismissal.
+- **Auto-dismiss**: Disappears automatically after a set duration.
+- **Hover Interactivity**: Auto-dismiss timer pauses on hover or focus, allowing users to interact with the content.
```

Comment:
[important] Let's place this description of the functionality of the Toast component in the `Behaviors`. Now we're repeating ourselves.

Reply:
Yeah, good idea